assets/js/mixitUp/style.css

```diff
@@ -1,200 +0,0 @@
-html,
```

Comment:
This was just a duplicate so I removed it.

Reply:
ok!